docs/tutorials/run-ollama-tutorial.md

```diff
@@ -0,0 +1,125 @@
+---
+sidebar_position: 2
+---
+
+# Run Ollama locally 
+
+<div>
+<img src={require("/static/img/llama.png").default} alt="Ollama"/>
+</div>
+
+
+In this tutorial we will see how to run OLLama
+using Fair Compute platform
+
+## Pull and run the OLlama Image
+
+Once you have rented a gpu and connected to the fair server, 
+run the following Docker command to pull and start the docker 
+container for ollama. 
+
+```bash
+fair docker run \      
+  --name ollama \
+  -p 11434:11434 \
+  ollama/ollama:latest
+
+```
+
+The fair docker command will automatically select the rented machine available
+without the need to specify the specific machine ID.
+
+Here's what this command does:
+1. fair docker run: Pulls image and starts the container.
+2. --name ollama : Specifies the name for the container.
+3. -p 11434:11434: Maps port 11434 on your server to port 11434 on the container,
+allowing you to access the Ollama.
+4. ollama/ollama:latest : Specifies the docker image to be run.
+
+After running this command, you'll see output similar to the following:
+
+```bash
+# b3501dc206e22f10e69e3c597d7a912423aa84dc665c0fc749b9c5a40c32632b
```

Comment:
Since you've removed the `-d` flag - the output will be different.

Reply:
It continues running the shell without the -d flag, so I have added it in the recent commit